README.md

```diff
@@ -30,7 +30,6 @@ If you have suggestions or problems, please [open an issue](https://github.com/n
 
 * **[👤 User system requirements](https://nextcloud-talk.readthedocs.io/en/latest/user-requirements/)**
 * **[📖 User documentation](https://docs.nextcloud.com/server/latest/user_manual/en/talk/index.html)**
-* **[📖 User documentation](https://docs.nextcloud.com/server/latest/user_manual/en/talk/index.html)**
```

Comment:
Ah, i guess it's a copy paste fail because the admin docs link is missing 🙈 but we don't have a good overview for that and the entry in the left sidebar is not clickable

https://nextcloud-talk.readthedocs.io/en/latest/#setup